gdk/gdk.go

```
@@ -646,6 +676,19 @@ func (v *Pixbuf) ScaleSimple(destWidth, destHeight int, interpType InterpType) (
 	return p, nil
 }
 
+// RotateSimple is a wrapper around gdk_pixbuf_rotate_simple().
+func (v *Pixbuf) RotateSimple(angle PixbufRotation) (*Pixbuf, error) {
+	c := C.gdk_pixbuf_rotate_simple(v.native(), C.GdkPixbufRotation(angle))
```

Right. So I think we should be panicking on a bad memory allocation (like everything else in Go) rather than returning an error to the caller.